.goreleaser.yml

```diff
@@ -115,11 +115,6 @@ dockers:
     - "--label=org.opencontainers.image.source={{ .GitURL }}"
     - "--platform=linux/arm64"
 docker_manifests:
-  - name_template: "stackexchange/{{.ProjectName}}:{{ .Version }}"
-    image_templates:
-      - *amd_image
-      - *386_image
-      - *arm_image
   - name_template: "stackexchange/{{.ProjectName}}:v{{ .Version }}"
```

Comment:
I usually see the non-v version used instead of v version:
1. [mariadb](https://hub.docker.com/_/mariadb/tags?page=1&name=v)
2. [nginx](https://hub.docker.com/_/nginx/tags?page=1&name=v)
3. [php](https://hub.docker.com/_/php/tags?page=1&name=v)

@tlimoncelli what is your reason for wanting to publish the v version?
- [dnscontrol](https://hub.docker.com/r/stackexchange/dnscontrol/tags?page=1&name=v)

Reply:
I guessed wrong :(

Reply:
More serious answer: I thought I should be consistent with the rest of the project. Now that I think about it, I should have looked to see what is typical for docker. 

I'll remove the "v".

Reply:
Fixed by https://github.com/StackExchange/dnscontrol/pull/2378.